hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/BaseFreonGenerator.java

```diff
@@ -266,14 +311,27 @@ public void init() {
         }, 10);
 
     executor = Executors.newFixedThreadPool(threadNo);
-
-    progressBar = new ProgressBar(System.out, testNo, successCounter::get,
-        freonCommand.isInteractive());
```

Comment:
Please keep the `isInteractive` flag.  Progress bar should not be console-based in non-interactive environment.  (Try `docker-compose exec -T scm ozone freon ...` for testing.)

Reply:
this has been restored.